src/Parser.js

```diff
@@ -853,6 +856,23 @@ export default class Parser {
     }
 
     /**
+     * Parses a url string.
+     */
+    parseUrlGroup(optional) {
+        const res = this.parseStringGroup("url", optional);
+        if (!res) {
+            return null;
+        }
+        const raw = res.text;
+        // hyperref package allows backslashes alone in href, but doesn't generate
+        // valid links in such cases; we interpret this as "undefiend" behaviour,
+        // and keep them as-is. In some environment, they're replaced by slashes
+        // in url by browser.
```

Comment:
Are the backslashes being replaced by forward slashes?  In which environments does this happen?

Reply:
At least in Safari 11.0.1 (13604.3.5) on macOS High Sierra 10.13.1.

Reply:
Ah... I was getting confused between LaTeX environments and browser environments.  Maybe reword that last sentence to be: `Some browser will replace backslashes with forward slashes.`

Reply:
Oh yes, it was really confusing expression. I will replace them.